src/app/shared/theme-chooser/theme-chooser.html

```diff
@@ -0,0 +1,16 @@
+<button md-icon-button [md-menu-trigger-for]="themeMenu">
+  <md-icon>format_color_fill</md-icon>
+</button>
+
+<md-menu class="theme-chooser-menu" #themeMenu="mdMenu" x-position="before">
```

Comment:
Tab should close the menu and the arrow keys should move between items. The menu only becomes focused if you opened it via keyboard, though. If I open the theme picker via keyboard, seems like focus never moves into the menu. Possible that it's a bug in `md-menu`. 

cc @kara